sdk/js/packages/client/src/model/model.ts

```diff
@@ -10,33 +10,33 @@ export interface AIChatFile {
   data: Uint8Array | File | Buffer;
 }
 
-export interface AIChatMessage {
+export interface AIChatMessage<TCtx extends object = object> {
```

Comment:
`object` is a restrictive type and doesn't allow access to any members. It is typically frowned upon to use it. Did you consider extending `Record<string, any>` instead?
```suggestion
export interface AIChatMessage<TCtx extends Record<string, any> = Record<string, any>> {
```

Reply:
Do you think we could use something a little more descriptive? It took me a second to figure out what `TCtx` was - how about `ContextType` ?

Reply:
Thank you for the feedback! I addressed `ContextType` rename, and made the default context type `Record<string, unknown>`